TSC-Charter.md

```diff
@@ -36,18 +36,19 @@ approved by the Board.
 
 ## Section 4. Establishment of the TSC.  
 
-TSC memberships are not time-limited. There is no fixed size of the TSC.
-However, the expected target is between 6 and 12, to ensure adequate
-coverage of important areas of expertise, balanced with the ability to
-make decisions efficiently.
+TSC memberships are not time-limited. There is no maximum size of the TSC.
+The size is expected to vary in order to ensure adequate coverage of important
+areas of expertise, balanced with the ability to make decisions efficiently.
+The TSC must have at least three members.
 
 There is no specific set of requirements or qualifications for TSC
 membership beyond these rules. The TSC may add additional members to the
-TSC by a standard TSC motion and vote.  A TSC member may be removed from
-the TSC by voluntary resignation, or by a standard TSC motion.
+TSC by a standard TSC motion and vote. A TSC member may be removed from the
+TSC by voluntary resignation, by a standard TSC motion, or in accordance to the
+attendance rules described below.
 
-Changes to TSC membership should be posted in the agenda, and may be
-suggested as any other agenda item.
+Changes to TSC membership should be posted in the agenda, and may be suggested
+as any other agenda item.
```

Comment:
Perhaps: *Nominations for* TSC membership should be... as any other agenda item *by any member of the community* (i.e. not just current TSC members).